python/servo/devenv_commands.py

```diff
@@ -143,109 +96,7 @@ def cargo_clippy(self, params, github_annotations=False, **kwargs):
 
-    @Command("grep", description="`git grep` for selected directories.", category="devenv")
-    @CommandArgument(
-        "params", default=None, nargs="...", help="Command-line arguments to be passed through to `git grep`"
-    )
-    def grep(self, params):
-        if not params:
-            params = []
-        # get all directories under tests/
-        tests_dirs = listdir("tests")
-        # Directories to be excluded under tests/
-        excluded_tests_dirs = ["wpt", "jquery"]
-        tests_dirs = filter(lambda dir: dir not in excluded_tests_dirs, tests_dirs)
-        # Set of directories in project root
-        root_dirs = ["components", "ports", "python", "etc", "resources"]
-        # Generate absolute paths for directories in tests/ and project-root/
-        tests_dirs_abs = [path.join(self.context.topdir, "tests", s) for s in tests_dirs]
-        root_dirs_abs = [path.join(self.context.topdir, s) for s in root_dirs]
-        # Absolute paths for all directories to be considered
-        grep_paths = root_dirs_abs + tests_dirs_abs
-        return call(
-            ["git"] + ["grep"] + params + ["--"] + grep_paths + [":(exclude)*.min.js", ":(exclude)*.min.css"],
-            env=self.build_env(),
-        )
-
     @Command("fetch", description="Fetch Rust, Cargo and Cargo dependencies", category="devenv")
     def fetch(self):
         self.ensure_bootstrapped()
         return call(["cargo", "fetch"], env=self.build_env())
-
-    @Command("ndk-stack", description="Invoke the ndk-stack tool with the expected symbol paths", category="devenv")
-    @CommandArgument("--release", action="store_true", help="Use release build symbols")
-    @CommandArgument("--target", action="store", default="armv7-linux-androideabi", help="Build target")
-    @CommandArgument("logfile", action="store", help="Path to logcat output with crash report")
-    def stack(self, release, target, logfile):
-        if not path.isfile(logfile):
-            print(logfile + " doesn't exist")
-            return -1
-
-        env = self.build_env()
-        ndk_stack = path.join(env["ANDROID_NDK"], "ndk-stack")
-        self.setup_configuration_for_android_target(target)
-        sym_path = path.join(
-            "target", target, "release" if release else "debug", "apk", "obj", "local", self.config["android"]["lib"]
-        )
-        print(subprocess.check_output([ndk_stack, "-sym", sym_path, "-dump", logfile]))
-
-    @Command("ndk-gdb", description="Invoke ndk-gdb tool with the expected symbol paths", category="devenv")
-    @CommandArgument("--release", action="store_true", help="Use release build symbols")
-    @CommandArgument("--target", action="store", default="armv7-linux-androideabi", help="Build target")
-    def ndk_gdb(self, release, target):
-        env = self.build_env()
-        ndk_gdb = path.join(env["ANDROID_NDK"], "ndk-gdb")
-        adb_path = path.join(env["ANDROID_SDK"], "platform-tools", "adb")
```

Comment:
currently unused, are you sure we wont need it ?

Reply:
> currently unused, are you sure we wont need it ?

They could be useful, but it's quite likely they will need to be reimplemented as they have totally bit rotted.